json.js

```diff
@@ -6,12 +6,15 @@ const jsonStringify = require('fast-safe-stringify');
 
 /*
  * function replacer (key, value)
- * Handles proper stringification of Buffer output.
+ * Handles proper stringification of Buffer and bigint output.
  */
 function replacer(key, value) {
-  return value instanceof Buffer
-    ? value.toString('base64')
-    : value;
+  if (value instanceof Buffer)
+    return value.toString('base64');
+  // eslint-disable-next-line valid-typeof
+  if (typeof value === 'bigint')
```

Comment:
`else if` here?

Reply:
Seemed cleaner without given that all `if`s are followed by `returns`. Are there linting rules you're using?

Reply:
Sorry for being pedantic, I changed my mind here, this is fine as-is.